### PR TITLE
Update install-wp-tests.sh and fix PHP tests on WP nightly

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue:

- #5593 

## Relevant technical choices

- `install-wp-tests.sh` updated to latest from wp-cli template https://raw.githubusercontent.com/wp-cli/scaffold-command/master/templates/install-wp-tests.sh

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: tests/phpunit/bin/install-wp-tests.sh [hunk 1]
```diff
@@ -166,7 +166,7 @@ install_db() {
 	fi
 
 	# create database
-	if [ $(mysql --user="$DB_USER" --password="$DB_PASS" --execute='show databases;' | grep ^$DB_NAME$) ]
+	if [ $(mysql --user="$DB_USER" --password="$DB_PASS"$EXTRA --execute='show databases;' | grep ^$DB_NAME$) ]
```
Comment (tofumatt) [inline tests/phpunit/bin/install-wp-tests.sh:169]: What's up with this addition? Is it from testing? I just don't see it mentioned elsewhere. 🤔 

```suggestion
	if [ $(mysql --user="$DB_USER" --password="$DB_PASS" --execute='show databases;' | grep ^$DB_NAME$) ]
```

**Review by tofumatt — COMMENTED** [review]
Looks good, just one question.

File: tests/phpunit/bin/install-wp-tests.sh (hunk 1 above)
Comment (aaemnnosttv) [reply tests/phpunit/bin/install-wp-tests.sh:169]: There could be weird stuff in there, it should be an exact copy of this: https://raw.githubusercontent.com/wp-cli/scaffold-command/master/templates/install-wp-tests.sh

I literally just dumped in the latest version of the file here 😄

Comment (tofumatt) [reply tests/phpunit/bin/install-wp-tests.sh:169]: Hah, okay cool. Works for me!

**Review by tofumatt — APPROVED** [review]

**Pull request merged**